pycc/ccwfn.py

```diff
@@ -305,10 +307,43 @@ def residuals(self, F, t1, t2):
 
         r1 = self.r_T1(o, v, F, ERI, L, t1, t2, Fae, Fme, Fmi)
         r2 = self.r_T2(o, v, F, ERI, L, t1, t2, Fae, Fme, Fmi, Wmnij, Wmbej, Wmbje, Zmbij)
-       
+             
         if isinstance(Fae, torch.Tensor):
-            del Fae, Fmi, Fme, Wmnij, Wmbej, Wmbje, Zmbij 
-
+            del Fae, Fmi, Wmnij, Wmbej, Wmbje, Zmbij 
```

Comment:
what happens to `Fme` if `Fae` is a `torch.Tensor`, but the model is something other than `CC3`?

Reply:
Yes, that should be changed. Thanks for catching that, Ben.